easybuild/easyconfigs/s/spglib/spglib-1.9.4.2-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "PythonPackage"
+
+name = 'spglib'
```

Comment:
also, the name `spglib`, instead of `spglib-python`, is needed for the source_url and for the sanity check, what's the best way to do that?